packages/dev/biome-config/configs/biome-base.json

```diff
@@ -0,0 +1,147 @@
+{
+	"$schema": "../node_modules/@biomejs/biome/configuration_schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"json": {
+		"formatter": {
+			"enabled": true,
+			"trailingCommas": "none",
+			"lineWidth": 100,
+			"indentStyle": "space",
+			"indentWidth": 2,
+			"lineEnding": "lf"
+		}
+	},
+	"javascript": {
+		"globals": ["vitest"],
+		"formatter": {
+			"enabled": true,
+			"arrowParentheses": "always",
+			"bracketSpacing": true,
+			"indentWidth": 2,
+			"indentStyle": "space",
+			"semicolons": "asNeeded",
+			"lineWidth": 100,
+			"quoteStyle": "single",
+			"trailingCommas": "all"
+		}
+	},
+	"files": {
+		"include": [
+			"./test/**/*.ts",
+			"./src/**/*.ts",
+			"./lib/**/*.ts",
+			"./index.ts",
+			"./scripts/**/*.ts",
+			"./biome.json",
+			"./package.json"
+		],
+		"ignore": ["./dist", "./coverage", "./node_modules"]
+	},
+	"formatter": {
+		"enabled": true,
+		"lineEnding": "lf",
+		"indentStyle": "space",
+		"indentWidth": 2
+	},
+	"linter": {
+		"rules": {
+			"performance": {
+				"noAccumulatingSpread": "error",
+				"noBarrelFile": "error",
+				"noDelete": "error",
+				"noReExportAll": "error"
+			},
+			"style": {
+				"noNonNullAssertion": "off",
+				"noDefaultExport": "error",
+				"noParameterAssign": "error",
```

Comment:
yes. personally I prefer explicit rule setup, but I'm open to remove default rules from the list

Reply:
I'm fine with it. I was just asking to make sure this is intentional.

My personal preference is to keep the defaults out. That way, I can look at the config and see only the subset that we decided to change. I suppose as Biome adds new rules to the recommended config, it will also get applied to our setup (even though it won't be explicitly specified here).

Reply:
I don't mind this approach.
is there a list of default rules I could use to remove them from here?

Reply:
Here is a list of [recommended rules](https://biomejs.dev/linter/rules/#recommended-rules).

Reply:
removed all of the default configuration